save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -275,7 +268,56 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
             tr {
                 th {
                     colSpan = tableInstance.columns.size
-                    testExecutionFiltersRow()
+                    testExecutionFiltersRow {
+                        status = state.status?.name ?: "ANY"
```

Comment:
It's better to pass field of state directly (then react will be able to track this relation) and only map data when required inside child component

Reply:
Done